banzai_floyds/utils/wavelength_utils.py

```diff
@@ -32,6 +32,7 @@ def to_header(self):
             header[f'POLYDOM{i + 1}'] = str(list(polynomial.domain))
             for j, coef in enumerate(polynomial.coef):
                 header[f'COEF{i + 1}_{j}'] = coef
+        header['EXTNAME'] = 'WAVELENGTHS'
```

Comment:
What is this header keyword? Extension name? Does this have any other values? I'm not familiar with how this is used elsewhere.

Reply:
This is a FITS standard keyword that defines the name of an extension. It allows you to index hdus with a name rather than just an integer, e.g. 
```
hdu_list = fits.open('foo.fits')
hdu_list['WAVELENGTHS'].data
```